config/alphanet/config.json

```
@@ -17,8 +17,14 @@
 		"communityIdentifier": "Lisk",
 		"maxTransactionsSize": 15360,
 		"minFeePerByte": 1000,
-		"baseFees": [],
-		"modules": {}
+		"modules": {
+			"reward": {
+				"tokenID": "0000000000000000",
+				"offset": 10,
```

Let us keep the `offset` similar to the current version. But let it remain `10` for `devnet`.
```suggestion
				"offset": 2160,
```